firefox-ios/Client/Frontend/Home/Homepage Rebuild/Bookmark/BookmarksCell.swift

```diff
@@ -55,14 +55,14 @@ final class BookmarksCell: UICollectionViewCell, ReusableCell, ThemeApplicable,
             cornerRadius: HomepageViewModel.UX.generalCornerRadius).cgPath
     }
 
-    func configure(state: BookmarkState, theme: Theme) {
+    func configure(item: BookmarkConfiguration, theme: Theme) {
```

Comment:
We named the first param `config` in most places - should we maybe use it here as well for consistency?

Reply:
I was torn on this because if we ever want to change config to something else it would require a lot of renaming.
 
For `PocketStandardCell.swift`, I didn't have to update the parameter names because `story` was applicable for both `state` or `config`, so I left it as-is.  For most things I renamed to `config` as the generic name, but thought I can improve in areas that I was more familiar with the context.  

Happy to make the change if you would prefer the consistency!

Reply:
Thanks @cyndichin. Maybe we can rename it the other areas when they get touched. In the end this is a nit and would just be great for consistency.

Reply:
sounds good, thanks for reviewing!